.scripts/localeOrderKeys.js

```diff
@@ -1,8 +1,7 @@
 const fs = require('fs');
 const path = require('path');
 const prettier = require('prettier');
-
-const localeDir = path.join(__dirname, '..', 'src', 'locale');
+const { localeDir, languages } = require('./utils.js');
 
 // order keys of a json object.
 const orderKeysAlphabetically = (o) => {
```

Comment:
This can be moved into `utils`.

Also, `isObject` (the below function) can also be moved into `utils`.